index.html

```diff
@@ -31,38 +31,42 @@ <h1 class="textdeco1 text1width">José Fernández-Caminero</h1>
     
             <p class="crazyParagraph">I love Full-Stack Engineering and I can code websites that are both the most complex and the most efficient.</p>
     
-            <button class="button1 width90">Start collaboration</button>
+            <button class="button1 width90 bigButton1">Start collaboration</button>
         </section>
     
         <div class="artdeco2"></div>
-    </div>
 
-    <div>
         <section class="dottedDeco"></section>
-    
+    </div>
+
+    <div class="topPadding">
         <section class="middleDeco"></section>
     </div>
 
-    <h2 class="textdeco2 margBottom projectsRotate">Projects</h2>
+    <h2 class="textdeco2 margBottom projectsrotate">Projects</h2>
```

Comment:
- [ ] I noticed here in your project section as well the heading is above the image so let us improve our design so I highly recommend to add more spacing here so kindly add some margin/padding to not be very close to the image it is very important to make your project identical to the figma as much as possible since it is one of the requirements ♻🚀🔥💪🏻😊📌

- Your design
![image](https://user-images.githubusercontent.com/58553711/201233778-0096e8cb-9e69-4935-b72c-923d4907b40e.png)

- Figma
![image](https://user-images.githubusercontent.com/58553711/201233823-82c3905d-c1f2-415c-9f0c-daf116724bca.png)